docker-container/README.md

```diff
@@ -0,0 +1,131 @@
+# Official Magento Documentation
+
+[https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html)
+
+# Dependencies
+
+## ngrok
+
+[https://ngrok.com/](https://ngrok.com/pricing)
+[https://ngrok.com/pricing](https://ngrok.com/pricing)
+
+In order for your docker container to communicate with the webhook and shipping & tax url set on [merchant.bolt.com](http://merchant.bolt.com/), one must set up an ngrok account to expose your store's docker ports to a public IP address. In order to do this one must have at least the Basic plan with ngrok to reserve a domain for themselves. 
+
+Once the initial configuration scripts are ran in your local environment and a public domain is reserved, run the following command 
+
+    ./ngrok http <STORE_PORT> -subdomain=<NGROK_DOMAIN>
+
+***Make sure not to include [ngrok.io](http://ngrok.io) in your domain***. For example if your store is [test-store.ngrok.io](http://test-store.ngrok.io), **just enter test-store** for the domain. 
+
+## Launch Script Dependencies
+
+### Python:
+
+Make sure you have a stable version of python in your environment. These scripts were ran with 2.7.15.
```

Comment:
Does this work or not work with python 3? It should be explicit in the `README`.